frontend/src/components/common/FormikPersist.tsx

```diff
@@ -11,59 +13,119 @@ export interface PersistProps {
   debounce?: number;
   isSessionStorage?: boolean;
   writeOnly?: boolean;
+  loadDraft?: boolean;
+}
+interface PersistState {
+  showLoadDraftDialog: boolean;
+}
+
+// Do not allow stored data to be loaded if the application version does not match.
+interface VersionedStorage {
+  version: string;
```

Comment:
How do we set this value?

Reply:
Currently I'm using the value set in package.json
All this means is that whenever we bump the version in a frontend release people will lose their drafts.

Reply:
I like this. Would love to see a comment somewhere for reference when we need it.

Reply:
Fair, will add in my next PR with the building updates.